tests/perf/cypress/e2e/Graphload.cy.js

```diff
@@ -0,0 +1,58 @@
+describe('graphload', () => {
+
+    http://localhost:3000/console/graph/namespaces/
+        // ?traffic=grpc%2CgrpcRequest%2Chttp%2ChttpRequest%2Ctcp%2CtcpSent
+        // &graphType=versionedApp
+        // &namespaces=bookinfo%2Ccert-manager
+        // &duration=60
+        // &refresh=15000
+        // &layout=kiali-dagre
+        // &namespaceLayout=kiali-dagre
+
+    var graphUrl;
+
+    before(() => {
+        cy.fixture('graphParams')
+            .then(function (data) {
+                graphUrl = encodeURI("/console/graph/namespaces?traffic="+data.traffic
+                    +"&graphType="+data.graphType
+                    +"&namespaces="+data.namespaces
+                    +"&duration="+data.duration
+                    +"&refresh="+data.refresh
+                    +"&layout="+data.layout
+                    +"&namespaceLayout="+data.namespaceLayout);
+            })
+            .as('data');
+    })
+
+    it('Visit Main Page', () => {
+        cy.visit("/")
+    })
+
+    it('Measure Graph load time', {
+            defaultCommandTimeout: Cypress.config('timeout')
+    }, () => {
+        cy.visit(graphUrl, {
+            onBeforeLoad(win) {
+                win.performance.mark("start");
+            }
+        })
+        .its("performance").then((performance) => {
+            cy.intercept(`**/api/namespaces/graph*`).as('graphNamespaces');
```

Comment:
You may need to move this call so that it is setup before the `cy.visit` call in case you try to visit the page before the intercept is setup.

Reply:
Moved. Thanks!